src/plutus/syntax.md

```diff
@@ -50,9 +50,3 @@ De Bruijn indices are used in serialized scripts.
 It therefore makes the most sense for a CEK machine implementation to use de Bruijn indices.
 
 When using de Bruijn indices, the binder `𝑥` in `(lam 𝑥 𝑀)` is irrelevant, and any number can be used.
```

Comment:
Zero would be a good choice!  I should check if we still do this, but we used to start our de Bruijn indices at 1, so 0 definitely didn't mean anything.

Reply:
Added "0 is a good choice since it is not a valid de Bruijn index."